module/builder/collection/builder.go

```diff
@@ -195,13 +195,29 @@ func (b *Builder) BuildOn(parentID flow.Identifier, setter func(*flow.Header) er
 		minRefID := refChainFinalizedID
 
 		var transactions []*flow.TransactionBody
+		var totalByteSize uint
+		var totalGas uint64
 		for _, tx := range b.transactions.All() {
 
 			// if we have reached maximum number of transactions, stop
 			if uint(len(transactions)) >= b.config.MaxCollectionSize {
 				break
 			}
 
+			// we can break cause
+			// max byte size per tx << the max collection byte size
```

Comment:
don't quite understand these two lines of comments

Reply:
I'll add more details